collector/generic_collector.go

```diff
@@ -83,12 +87,18 @@ func NewCollector(collectorName string, configFile []byte) (*GenericCollector, e
 		minPollFrequency = minSupportedFrequency
 	}
 
+	if len(configInJSON.MetricsConfig) > metricCountLimit {
+		return nil, fmt.Errorf("Too many metrics defined: %d limit: %d", len(configInJSON.MetricsConfig), metricCountLimit)
```

Comment:
Yeah. My comment applies to both the collectors. As of now, we will not collect any metrics for a container with a application metrics configuration that includes more than 100 metrics. 
My gut feeling is that we might want to skip application metrics and continue to collect the rest of the metrics like cpu, memory, network, etc. WDYT?